### PR TITLE
Allow running blivet-gui in Anaconda installer

### PR DESCRIPTION
**Pull request by vojtechtrefny**
First batch of patches allowing to run blivet-gui in Anaconda installer.

### CODE REVIEW

File: blivetgui/exception_handler.py [hunk 1]
```diff
@@ -67,14 +68,15 @@ def handle_exception(self, exc_type, exc_value, exc_traceback):
         else:
             tr_str = "".join(traceback.format_tb(exc_traceback))
 
-        if self.allow_ignore:
+        allow_report = command_exists("gnome-abrt")
```
Comment (M4rtinK) [inline blivetgui/exception_handler.py]: What about reporting exceptions when running Anaconda ? Is that also handled by this or not yet/elsewhere ?

Comment (vojtechtrefny) [inline blivetgui/exception_handler.py]: This exception handler is not used when running from Anaconda. Default exception handler (python-meh) should handle all exceptions in that case.

File: blivetgui/communication/errors.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#
+# Red Hat Author(s): Vojtech Trefny <vtrefny@redhat.com>
+#
+# ---------------------------------------------------------------------------- #
+
+
+class CommunicationError(Exception):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```
Comment (vpodzime) [inline blivetgui/communication/errors.py]: this has no effect

Comment (vojtechtrefny) [inline blivetgui/communication/errors.py]: fixed

**Pull request merged**